cmd/snap/cmd_search.go

```diff
@@ -0,0 +1,88 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2016 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package main
+
+import (
+	"fmt"
+	"os"
+	"sort"
+	"text/tabwriter"
+
+	"github.com/ubuntu-core/snappy/client"
+	"github.com/ubuntu-core/snappy/i18n"
+	"github.com/ubuntu-core/snappy/logger"
+)
+
+var (
+	shortSearchHelp = i18n.G("Search for packages to install")
+	longSearchHelp  = i18n.G("Query the store for available packages")
+)
+
+type cmdSearch struct {
+	Positional struct {
+		Query string `positional-arg-name:"query"`
+	} `positional-args:"yes"`
+}
+
+func init() {
+	cmd, err := parser.AddCommand("search", shortSearchHelp, longSearchHelp, &cmdSearch{})
+	if err != nil {
+		logger.Panicf("unable to add search command: %v", err)
+	}
+
+	cmd.Aliases = append(cmd.Aliases, "se")
```

Comment:
We can replace it by "find" if typing is the problem.. but sounds awkward to encourage people to use "snap se foo".